azure-pipelines-pr-template.yml

```diff
@@ -0,0 +1,95 @@
+# Runs for every pull request on master
```

Comment:
rename this file to `azure-pipelines-pr-test.yml`

Reply:
i'll name it to that when i build another pipeline in master with designer. this is just a temporary file